packages/backend/src/apps/odoo/common/xmlrpc-client.ts

```diff
@@ -0,0 +1,67 @@
+import { join } from 'node:path';
+import xmlrpc from 'xmlrpc';
+import { IGlobalVariable } from "@automatisch/types";
+
+type AsyncMethodCallPayload = {
+  method: string;
+  params: any[];
+  path?: string;
+}
+
+export const asyncMethodCall = async <T = number>($: IGlobalVariable, { method, params, path }: AsyncMethodCallPayload): Promise<T> => {
+  return new Promise(
+    (resolve, reject) => {
+      const client = getClient($, { path });
+
+      client.methodCall(
+        method,
+        params,
+        (error, response) => {
+          if (error != null) {
+            // something went wrong on the server side, display the error returned by Odoo
+            reject(error);
+          }
+
+          resolve(response);
+        }
+      )
+    }
+  );
+}
+
+export const getClient = ($: IGlobalVariable, { path = 'common' }) => {
```

Comment:
Maybe it would easier if the default was 'object' as it would be used more often that 'common'. 

I don't feel strongly about this, if you want to leave this as is, fine with me.

Reply:
Thanks a bunch for your remark. For now, I'll leave it as-is, as this can be easily iterated later without requiring any migration. I'll go ahead and merge this integration into the main branch then. 🎉